fh_immuta_utils/client.py

```diff
@@ -519,6 +522,24 @@ def tag_data_source(self, id: int, tag_data: List[Dict[str, Any]]) -> bool:
         res.raise_for_status()
         return True
 
+    def get_remote_database_test_response(
+        self, dataset_spec: Dict[str, Any]
+    ) -> requests.Response:
+        """
+        Tests a remote database enrollment.
+        :param dataset_spec: dataset configuration dictionary
+        :return: test response
+        """
+        request_prefix = blob_handler_type(dataset_spec["handler_type"])
+        remote_database = dataset_spec["database"]
+        headers = self.make_generic_odbc_request_headers(dataset_spec)
+        path = f"{request_prefix}/database/{remote_database}/test"
+        res = self._session.get(path, headers=headers)
+        return res
```

Comment:
what is the reason for adding this test here?

Reply:
This makes an API call to run a test on a remote database connection in Immuta. It tells us some simple things like when this remote database connection was created, if it's active, but most importantly for this diff, it also returns information about schema monitoring for that remote database. So this is the call that let's us know if schema monitoring is currently enabled or not.